tests/test_browser.py

```
@@ -11,45 +9,45 @@ def test_session_created():
-    result = asyncio.run(session.fill("#inp", "value"))
+    result = await session.fill("#inp", "value")
     assert result.success is False
 
 
-def test_stub_mode_screenshot():
+async def test_stub_mode_screenshot():
     session = BrowserSession()
     if session.available:
         pytest.skip("Playwright installed")
-    result = asyncio.run(session.screenshot("/tmp/snap.png"))
+    result = await session.screenshot("/tmp/snap.png")
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/snap.png"

(S108)